src/configurations/destinations/userpilot/ui-config.json

```diff
@@ -28,24 +28,6 @@
                       ]
                     }
                   },
-                  {
```

Comment:
Is this not mandatory field if connection setup in device mode ?

Reply:
no, it is an optional config that allows loading userpilot sdk from a different url, not mandatory